src/macaron/output_reporter/templates/base_template.html

```diff
@@ -255,26 +255,30 @@
             padding-left: 2em
         }
 
-        .caret {
+        /*
+            The reason why we need to create a separate .togler class is because all .caret calss are set binded
```

Comment:
Typos:
`.togler` -> `.toggler`
`calss` -> `class`

Reply:
Fixed in https://github.com/oracle/macaron/pull/165/commits/3a18c96a78a3da6c6cea8d18a784cdaf0dd7bb9d